guides/source/getting_started.md

```diff
@@ -242,262 +241,740 @@ different _actions_. Each action's purpose is to collect information to provide
+`db/migrate/[timestamp]_create_articles.rb`.
+
+```ruby
+class CreateArticles < ActiveRecord::Migration[6.0]
+  def change
+    create_table :articles do |t|
+      t.string :title
+      t.text :body
+
+      t.timestamps
+    end
+  end
 end
 ```
 
-If you run `bin/rails routes`, you'll see that it has defined routes for all the
-standard RESTful actions.  The meaning of the prefix column (and other columns)
-will be seen later, but for now notice that Rails has inferred the
-singular form `article` and makes meaningful use of the distinction.
+This file contains Ruby code to create a table within our application's
+database. Migrations are written in Ruby so that they can be database-agnostic
+-- regardless of what database you use with Rails, you'll always write
+migrations in Ruby.
+
+Inside this migration file, there's a `create_table` method that defines how
+the `articles` table should be constructed. This method will create a table in
+our database that contains an `id` auto-incrementing primary key. That means
+that the first record in our table will have an `id` of 1, and the next `id` of
+2, and so on.  Rails assumes by default this is the behaviour we want, and so
+it does this for us.
+
+Inside the block for `create_table`, we have two fields, `title` and `body`.
+These were added to the migration automatically because we put them at the end
+of the `rails g model` call:
 
 ```bash
-$ bin/rails routes
-       Prefix Verb   URI Pattern                  Controller#Action
-welcome_index GET    /welcome/index(.:format)     welcome#index
-     articles GET    /articles(.:format)          articles#index
-              POST   /articles(.:format)          articles#create
-  new_article GET    /articles/new(.:format)      articles#new
- edit_article GET    /articles/:id/edit(.:format) articles#edit
-      article GET    /articles/:id(.:format)      articles#show
-              PATCH  /articles/:id(.:format)      articles#update
-              PUT    /articles/:id(.:format)      articles#update
-              DELETE /articles/:id(.:format)      articles#destroy
-         root GET    /                            welcome#index
+$ bin/rails g model article title:string body:text
 ```
 
-In the next section, you will add the ability to create new articles in your
-application and be able to view them. This is the "C" and the "R" from CRUD:
-create and read. The form for doing this will look like this:
+On the last line of the block is `t.timestamps`. This method defines two
+additional fields in our table, called `created_at` and `updated_at`. When we
+create or update model objects, these fields will be set respectively.
+
+The structure of our table will look like this:
+
+| id | title  | body | created_at | updated_at |
+|----|------- |------|------------|------------|
+|    | &nbsp; |      |            |            |
+
+
+To create this table in our application's database, we can run this command:
+
+```bash
+rails db:migrate
```

Comment:
The rails command should be written with `bin/rails`:
`$ bin/rails db:migrate`
Adding the $ also makes it look better with the code highlighter.